docs/components/modeler/web-modeler/connectors/available-connectors/slack.md

```diff
@@ -0,0 +1,92 @@
+---
+id: slack
+title: Slack connector
+description: Working with Slack connector
+---
+
+The **Slack connector** allows you to send messages to channels or users in your [Slack](https://slack.com) workspace from your BPMN process.
+
+## Prerequisites
+
+To use the Slack connector, a Slack app must be registered with the Slack workspace you would like to send messages to. A respective OAuth token needs to be configured as a secret in your cluster. Follow [these steps in the appendix](#appendix) to learn how to set this up.
+
+## Create a Slack connector task
+
+To use a **Slack connector** in your process, either change the type of an existing task using the **change type** context menu, or create a new connector task by using the **append** context menu. Follow [our guide on using connectors](../use-connectors.md) to learn more.
+
+## Make your Slack connector executable
+
+To make the **Slack connector** executable, fill out the mandatory fields highlighted in red in the properties panel:
+
+![slack connector red properties](../img/connectors-slack-red-properties.png)
+
+1. Set **Authentication** to your Slack OAuth token, which is stored as a secret. For example 'secrets.SLACK_OAUTH_TOKEN`
+2. Set **Channel/User Name** to either the **channel** or **user** you want to send the message to
+   1. A **channel** is specified by an unique identifier starting with an `#` (example: `#myChannel`)
+   2. A **user** is specified by a user name starting with an `@` (example: `@myUser`)
+3. Set **Message** to the message string you would like to send (example: `Hello World!`)
+
+The **Channel/User Name** and **Message** can either be given static values, or FEEL expressions. FEEL expressions can be used to [access process variables or dynamically create values](../../../../concepts/expressions.md). This can be handy if for example a process variables is used to store the relevant channel or if the message needs to be composed dynamically:
+
+![slack connector compose](../img/connectors-slack-compose.png)
+
+## Slack API response
+
+The Slack API response will be available in a temporary local `response` variable.
```

Comment:
Again just a proposal.


I'd remove any qualifier such as "temporary" (because that raises more questions than it answeres). Instead I'd coin the term _expose a variable_ (that I can either save or extract fields from).


Example: "The connector exposes the SLACK API result as a variable called `response` with the following fields: ...

In the output variable section you can configure how to use this variable:

..."

Reply:
yes, I buy that. Adjusted

Reply:
Although I think `local` is important. I did link to an explanation of what variable scopes are.